design/one-pager-package-runtime-config.md

```diff
@@ -0,0 +1,222 @@
+# Package Runtime Config
+
+* Owner: Nic Cope (@negz)
+* Reviewers: TODO
```

Comment:
Will figure out some explicit reviewers ASAP.

Reply:
@hasheddan @turkenh do you have bandwidth to review this?

Reply:
@turkenh confirmed he does. @hasheddan I'm hoping you do too since you're passionate about this topic. LMK if you want me to remove you. 😃